bin/collator/src/command.rs

```diff
@@ -819,9 +819,20 @@ pub fn run() -> Result<()> {
             let runner = cli.create_runner(&cli.run.normalize())?;
             let collator_options = cli.run.collator_options();
 
+            let tracing_config = crate::cli::EvmTracingConfig {
```

Comment:
Please let's call this `evm_tracing_config` to make it more clear.

`tracing_config` sounds too generic.